config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1 @@
+StrongMigrations.start_after = "20190518115041".to_i
```

Comment:
Why do this as a string and then convert it to integer? Why not just write it as an integer?

Reply:
It’s a hacky way to avoid rubocop complaining about underscores in large integers  🙂 perhaps I should be more sensible and use a magic rubocop disable-comment instead.

Reply:
Oh I thought I had turned that off because I kind of hate it myself ;-)

Reply:
Oh, I'm fine with the underscores for actual numbers (like 50_000), but this string of digits isn't really representing 20 trillion, 190 billion, 518 million and so on somethings, so the underscores are meaningless here - e.g. 20_190_518_115_041. 

Anyway, I've used a rubocop disable comment instead, which is probably the best approach.

Reply:
Oh I was going to just merge it as is, just didn't get around to it last night. I'm not a great fan of disable comments personally.